cips/cip-1.md

```diff
@@ -316,6 +311,8 @@ format. There is a [CIP template](./cip-template.md) to follow.
 Each CIP must begin with an RFC 822 style header preamble, preceded
 and followed by three hyphens (---). This header is also termed
 “front matter” by Jekyll. The headers must appear in the following order.
+In order to display on the CIP site, the frontmatter must also
+be surrounded by three backticks (```).
```

Comment:
[nit] I think the approach taken in this PR is to use a markdown table so perhaps this needs to be updated.

```suggestion
In order to display on the CIP site, the frontmatter must be
formatted in a markdown table.
```